packages/react-components/components/PhoneNumberWithFlag.tsx

```diff
@@ -22,8 +23,8 @@ export class PhoneNumberWithFlag extends React.PureComponent<Props> {
               )
             : getCountryEmoji(this.props.e164PhoneNumber)}
         </Text>
-        <Text style={fontStyles.telephoneHeadline}>
-          {(parsedNumber && parsedNumber.displayNumber) || ''}
+        <Text style={[fontStyles.regular, style.phoneNumber]}>
+          {(parsedNumber && `+${parsedNumber.countryCode} ${parsedNumber.displayNumber}`) || ''}
```

Comment:
Nit: I would find this more readable as `parsedNumber ? `+${parsedNumber.countryCode} ${parsedNumber.displayNumber}`) : ''`